tensorflow_tts/processor/ljspeech.py

```diff
@@ -11,6 +11,8 @@
 from tensorflow_tts.utils import cleaners
 from tensorflow_tts.processor import BaseProcessor
 
+from dataclasses import dataclass
+
 valid_symbols = [
```

Comment:
Let's load it from file maybe :D 
```python3
with open("prepro/SOMEPATH", "r") as f:
    valid_symbols = f.readlines()
```

Or save all symbols and mapper to json file and load it using processor class 📦 also uploading it to repo